colbert/indexing/codecs/residual_embeddings.py

```diff
@@ -29,28 +28,50 @@ def load_chunks(cls, index_path, chunk_idxs, num_embeddings):
 
         dim, nbits = get_dim_and_nbits(index_path)
 
-        codes = torch.empty(num_embeddings, dtype=torch.int32)
-        residuals = torch.empty(num_embeddings, dim // 8 * nbits, dtype=torch.uint8)
+        print_message("#> Loading codes and residuals...")
 
-        codes_offset = 0
+        if len(chunk_idxs) == 1: # Memory mapping is enabled for single-file indexes
```

Comment:
Can you add comments explaining what the constants are for? In particular, the 80 and 320

Reply:
Done